TestAssets/TestProjects/FilterProfiles/NewtonsoftMultipleVersions.xml

```diff
@@ -0,0 +1,4 @@
+<CacheArtifacts>
+  <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+  <PackageReference Include="Newtonsoft.Json" Version ="9.0.2-beta2"/>
```

Comment:
I don't want to start doing something different just because we can.
I would stick to conventions as much as possible